.gitignore

```diff
@@ -1,2 +1,3 @@
-composer.lock
```

Comment:
Je me suis renseigné d'avantage sur le fichier `composer.lock`.
Si ce fichier est présent, lorsqu'un projet qui dépend de ce paquet utiliser composer, le fichier `composer.lock` utilisé est celui du projet et non des dépendances.
Le fichier `composer.lock` est utilisé uniquement lorsque composer est lancé à partir du dossier "racine" du paquet.

Ceci dit, le fichier peut totalement être commit puisque ça permet de réaliser les tests unitaires automatisés (TravisCI par ex.) avec les mêmes versions des dépendances installés durant le développement et aussi permettre de mieux repérer les problèmes liés aux versions des dépendances.

A voir donc, s'il faut effectivement l'ajouter, ou non :p

Source : http://stackoverflow.com/questions/18901094/composer-lock-in-library